test/response.test

```diff
@@ -90,15 +90,15 @@ namespace eval ::qcode::test {
 <div class="message"></div>
 <div class="record"></div>
 <div class="action"></div>
```

Comment:
Could we maybe have something like this for object-property elements ... will call you in a bit to discuss:

```
<div class="object-property" data-name="surname" data-value="O'Neil" data-type="string"</div>
```